nyaa/api_handler.py

```diff
@@ -50,6 +50,22 @@ def decorator(*args, **kwargs):
 
 # #################################### API ROUTES ####################################
 
+    if upload_request.method == 'POST' and form.validate():
+        torrent = backend.handle_torrent_upload(form, user, True)
+
+        return flask.make_response(flask.jsonify({'Success': int('{0}'.format(torrent.id))}), 200)
+    else:
+        return_error_messages = []
+        for error_name, error_messages in form.errors.items():
+            return_error_messages.extend(error_messages)
+
+        return flask.make_response(flask.jsonify({'Failure': return_error_messages}), 400)
+
```

Comment:
Is this intentional/correct? 'Cause the code is unreachable as is (see the `return` statement a few lines above for context).